src/components/Navbar.tsx

```diff
@@ -1,33 +1,63 @@
 import { Navbar, Container, Nav } from 'react-bootstrap'
 import { Link, Outlet } from 'react-router-dom'
-
+import {Dropdown} from 'react-bootstrap';
 export default function MyNavbar() {
   return (
+    
     <>
-      <Navbar expand="lg" bg="light">
+      <Navbar expand="lg"  className = "nav-bar">
         <Container>
           <Navbar.Brand>
-            <Link className="nav-link" to="/">
-              BookBix
+            <Link className="nav-link" to="/" >
+              <p className = "bookbix-logo-org">BookBix<sup className = "sup-script">org</sup></p>
             </Link>
           </Navbar.Brand>
-          <Navbar.Toggle aria-controls="basic-navbar-nav" />
-          <Navbar.Collapse id="basic-navbar-nav">
-            <Nav className="ms-auto">
-              <Link className="nav-link" to="/profile">
-                Profile
-              </Link>
-              <Link className="nav-link" to="/register">
-                Register
-              </Link>
-              <Link className="nav-link" to="/login">
-                Login
-              </Link>
-            </Nav>
-          </Navbar.Collapse>
-        </Container>
-      </Navbar>
-      <Outlet />
-    </>
+          <Dropdown className='dropdown' >
+        <Dropdown.Toggle id="dropdown-autoclose-true">
+          Sorawit
```

Comment:
If you want Navbar to display Username instead a profile picture, so you can't hard code with "Sorawit".